examples/website/src/shared/molecules/commentTemplate.tsx

```diff
@@ -0,0 +1,427 @@
+    }
+  })
+
+  return (
+    <div className={role === 'reply' ? css(ReplyBox) : css(Container)}>
+      {header ? <h3>{header}</h3> : ''}
+      <form
+        onSubmit={e => {
+          e.preventDefault()
+          addComment({
+            variables: {
+              input: {
+                parentID,
+                text: commentInput,
+                itemID,
+                itemType
+              }
+            }
+          })
+        }}>
+        {commentState && <p className={css(StateMessage)}>{commentState}</p>}
+        <div className={css(CommentInput)}>
+          <RichTextBlock
+            value={commentInput}
+            onChange={input => {
+              setCommentInput(input)
+            }}
+          />
+        </div>
+        <p>
+          <BaseButton
+            css={props.parentCommentAuthor ? SmallButton : CommentButton}
+            disabled={loading}>
+            {props.parentCommentAuthor ? 'Submit' : 'Post comment'}
+          </BaseButton>
+        </p>
+      </form>
+    </div>
+  )
+}
+
+export function DisplayComments(props: DisplayCommentsProps) {
+  const css = useStyle()
+
+  const [commentID, setCommentID] = useState('')
+  const {comments} = props
+
+  return (
+    <div className={css(Container, CommentBox)}>
+      <h3 style={{marginTop: '2em'}}>Showing all comments</h3>
+      {comments?.map(parentComment => (
+        <ParentComment
+          comment={parentComment}
+          key={parentComment.id}
+          handleCurrentComment={(commentID: React.SetStateAction<string>) =>
+            setCommentID(commentID)
+          }
+          activeComment={commentID}
+        />
+      ))}
+    </div>
+  )
+}
+
+function ParentComment(props: any) {
+  const css = useStyle()
+
+  const {
+    id,
+    state,
+    rejectionReason,
+    userName,
+    authorType,
+    modifiedAt,
+    children,
+    text,
+    itemID,
+    itemType
+  } = props.comment
+
+  // This allows to display the children in a chronological order for the sake of a better UX
+  let childrenReversed = [...children].reverse()
+
+  const {activeComment} = props
+
+  return (
+    <div className={css(Thread)}>
+      <div className={css(CommentBox)} id={id}>
+        <a href={'#' + id} className={css(commentBorderLink)}>
+          <span className={css(SrOnly)}>Jump to comment {id}</span>
+        </a>
+        <div className={css(CommentHeading)}>
+          <div style={{width: 50}}>
+            <Image src={'../../../static/icons/avatar.jpg'} />
+          </div>
+          <div className={css(RightColumn)}>
+            <h4 className={css(CommentAuthor)}>{userName}</h4>
+            <p className={css(CommentMeta)}>
+              <span>{authorType}</span> ·{' '}
+              <span className={css(Timestamp)}>{new Date(modifiedAt).toLocaleString()}</span>
+            </p>
+          </div>
+        </div>
+
+        <div className={css(CommentBody)}>
+          <RichTextBlock disabled value={text} onChange={value => value} />
```

Comment:
What is happening inside `onChange`? If you want to make a function do nothing at all you can write `() => { /* do nothing*/ }`

Reply:
done